Modules/CodeTrek/Resources/views/evaluate.blade.php

```diff
@@ -79,8 +79,8 @@ class="collapse {{ $applicantDetail->id ? ($loop->last ? 'show' : '') : '' }}"
                                  @csrf
                                  <button type="submit" name="action" value="completed" class="btn btn-dark ml-2">Mark
                                      Completed</button>
-                                 <button type="submit" name="action" value="inactive" class="btn btn-danger ml-1">Mark
-                                     Inactive</button>
+                                <button type="submit" name="action" value="{{$applicant->status == "active"? "inactive" : "active"}}" class="btn btn-danger ml-1">
+                                    Mark {{ucfirst( $applicant->status == "active" ? "inactive" : "active")}} </button>  
```

Comment:
Minor formatting issues. Please take care of these things
```suggestion
                                    Mark {{ ucfirst($applicant->status == "active" ? "inactive" : "active") }} </button>  
```